internal/config/config_test.go

```diff
@@ -508,36 +1200,79 @@ func TestToRawYaml(t *testing.T) {
 		return nil
 	}
 	tests := []test{
-		// TODO test cases
-		/*
-		   {
-		       name: "test_case_1",
-		       args: args {
-		           data: nil,
-		       },
-		       want: want{},
-		       checkFunc: defaultCheckFunc,
-		   },
-		*/
-
-		// TODO test cases
-		/*
-		   func() test {
-		       return test {
-		           name: "test_case_2",
-		           args: args {
-		           data: nil,
-		           },
-		           want: want{},
-		           checkFunc: defaultCheckFunc,
-		       }
-		   }(),
-		*/
+		{
+			name: "return row string when data is an int type",
+			args: args{
+				data: 1,
+			},
+			want: want{
+				want: "1\n",
+			},
+		},
+		{
+			name: "return row string when data is a string type",
+			args: args{
+				data: "vdaas.vald",
+			},
+			want: want{
+				want: "vdaas.vald\n",
+			},
+		},
+		{
+			name: "return row string when data is a map string type",
+			args: args{
+				data: map[string]string{
+					"time_zone": "UTC",
+				},
+			},
+			want: want{
+				want: "time_zone: UTC\n",
+			},
+		},
+		{
+			name: "return row string when data is a nested map type",
+			args: args{
+				data: map[string]interface{}{
+					"logging": map[string]interface{}{
+						"logger": "glg",
+					},
+				},
+			},
+			want: want{
+				want: "logging:\n  logger: glg\n",
+			},
+		},
+		{
+			name: "return row string when data is a empty string",
+			args: args{
+				data: "",
+			},
+			want: want{
+				want: "\"\"\n",
+			},
+		},
+		{
+			name: "return row string when data is a GlobalConfig type",
+			args: args{
+				data: GlobalConfig{
+					Version: "v1.0.0",
+					TZ:      "UTC",
+					Logging: &Logging{
+						Logger: "glg",
+						Level:  "warn",
+						Format: "json",
+					},
+				},
+			},
+			want: want{
+				want: "version: v1.0.0\ntime_zone: UTC\nlogging:\n  logger: glg\n  level: warn\n  format: json\n",
+			},
+		},
```

Comment:
Can you add the flowing case?
- when data is nil

Reply:
thank you for your comment.  :bow:
I added new test case.